tools/ci/scripts/createnugetpackages.ps1

```diff
@@ -0,0 +1,180 @@
+<#
```

Comment:
This file seems unrelated to QR codes, and also maybe unfinished (the actual nuget creation seems commented out). Should this be moved to a different PR?

Reply:
id like to keep this file, it lets us validate the QR Code references are resolving correctly

Reply:
Sounds reasonable, maybe just add a TODO explaining the commented part if we plan to make it work eventually.